script.js

```diff
@@ -0,0 +1,76 @@
+const booksListContainer = document.querySelector(".books-list");
+const form = document.querySelector(".form");
+
+let books = [];
+if (localStorage.getItem("books")) {
+  books = JSON.parse(localStorage.getItem("books"));
+}
+
+function loadItems(booksArr) {
+  const booksList = booksArr.map(
+    (bk) => `<li class="book">
+<p>${bk.title}</p>
+ <p>${bk.author}</p>
+ <button class="btn btn-remove" data-id="${bk.id}">Remove</button>
+<hr />
+</li>`,
+  );
+
+  booksListContainer.innerHTML = booksList;
+}
+
+loadItems(books);
+
+function appendBook(bookObj) {
+  const listItem = `<li class="book">
+<p>${bookObj.title}</p>
+ <p>${bookObj.author}</p>
+ <button class="btn btn-remove" data-id="${bookObj.id}">Remove</button>
```

Comment:
- [ ] I was unable to check the book removal since I was unable to add a book in the first place. Kindly ensure that when the remove button is clicked the book is removed from the page and local storage